packages/ChangelogLinker/src/Console/ChangelogConsoleApplication.php

```diff
@@ -26,12 +28,17 @@ public function setRequiredDependencies(ParameterProvider $parameterProvider): v
         $this->parameterProvider = $parameterProvider;
     }
 
-    public function doRun(InputInterface $input, OutputInterface $output): int
+    protected function doRunCommand(Command $command, InputInterface $input, OutputInterface $output): int
     {
-        $input->bind($this->getDefinition());
+        // required to merge application + command definitions
+        $command->mergeApplicationDefinition();
+
+        $input = new ArgvInput();
+        $input->bind($command->getDefinition());
```

Comment:
A tip from: https://matthiasnoback.nl/2013/11/symfony2-add-a-global-option-to-console-commands-and-generate-pid-file/

To work with options/argument before running the command itself.